src/subcommand/server.rs

```diff
@@ -740,6 +741,29 @@ impl Server {
     })
   }
 
+  async fn runes_balances(Extension(index): Extension<Arc<Index>>) -> ServerResult {
```

Comment:
It should have the AcceptJson extension in the function arguments, so that it can be disabled with `--disable-json` flag.

Reply:
And also it should return a 404 if the client does not set the JSON application header